src/FOMObot/Helpers/Bot.hs

```diff
@@ -56,3 +63,21 @@ alertFOMOChannel channelID = do
     Slack.sendMessage fomoChannel message
   where
     message = "Check out <#" <> (channelID ^. Slack.getId) <> ">"
+
+processCommand :: Slack.Event -> Bot ()
```

Comment:
ya, I've been feeling this too ... I think it needs to be broken up ... I'm also not a huge fan of everything being a "helper"